### ISSUE TITLE
How to work with this repository?

### ISSUE THREAD

**Issue by zse4vfr** [NVlabs/I2SB#20]
Hi! I have been trying to launch this repository for 2 months. I could install for windows and for linux it by: 
    conda env create --file requirements.yaml python=3
    conda activate i2sb

I tried to run this command:  
    python sample.py --ckpt $NAME --n-gpu-per-node $N_GPU \
    --dataset-dir $DATA_DIR --batch-size $BATCH --use-fp16 \
    [--nfe $NFE] [--clip-denoise]

My colleague tried to use this one:
    python train.py --name $NAME --n-gpu-per-node $N_GPU \
    --corrupt $CORRUPT --dataset-dir $DATA_DIR \
    --batch-size $BATCH --microbatch $MICRO_BATCH [--ot-ode] \
    --beta-max $BMAX --log-dir $LOG_DIR [--log-writer $LOGGER]
    
Everything didn't work. Can you write to my gmail  "zse4vfr@gmail.com" and explain what should I do from very start to finish, please? I mean dependencies, sequencing, etc. Thank you!

**Comment by paomian001** [comment]
UPUPUP

**Comment by paomian001** [comment]
UPUPUP!

**Comment by zse4vfr** [comment]
Dear author! Can you help me before tomorrow, please?

**Comment by tongchangD** [comment]
cloud you solved this issue?

**Comment by zse4vfr** [comment]
no, i couldn't. it was my task for laboratory work, however after my course was done i finished my tries
